ch04_Operationeel.md

```diff
@@ -83,46 +82,72 @@ Het vaststellen van een nieuwe release van afzonderlijke Digikoppeling onderdele
+   Digikoppeling-onderdelen.
 5. De Digikoppeling-beheerder zorgt voor de voorbereiding van de
-releaseplanning.
-6. Het tactisch overleg beoordeelt de
-releasevoorstellen en stelt het beleid en de roadmap van
-nieuwe releases van de Digikoppeling-standaard vast in het
-releaseplanningsproces.
+   releaseplanning.
+6. Het tactisch overleg beoordeelt de releasevoorstellen en stelt
+   het beleid en de roadmap van nieuwe releases van de
+   Digikoppeling-standaard vast in het releaseplanningsproces.
 7. Bij het vaststellen van de inhoud van een nieuwe release van een
-Digikoppeling onderdeel wordt gestreefd naar consensus. Als consensus
-uitblijft zal de Digikoppeling beheerder, samen met het Ministerie
-van BZK de inhoud van een nieuwe release vaststellen.
+   Digikoppeling onderdeel wordt gestreefd naar consensus. Als consensus
+   uitblijft zal de Digikoppeling beheerder, samen met het Ministerie
+   van BZK de inhoud van een nieuwe release vaststellen.
 8. Bij het vaststellen van een nieuwe release zal het strategisch overleg
-uitspraken doen over het ondersteunen van oude releases.
+   uitspraken doen over het ondersteunen van oude releases.
 9. Maximaal kunnen twee (opéénvolgende) releases van een Digikoppeling
-onderdeel gelijktijdig de status „In Gebruik‟ hebben.
+   onderdeel gelijktijdig de status „In Gebruik‟ hebben.
 10. De releasetermijnen voor de verschillende Digikoppeling-onderdelen
-zijn afgestemd op de omgeving waarin deze worden gebruikt. Koppelvlak
-standaarden hebben bijvoorbeeld een kortere releasetermijn dan de
-bovenliggende architectuur.
-11. In bijzondere gevallen kan van de releasetermijn worden
-afgeweken.
+   zijn afgestemd op de omgeving waarin deze worden gebruikt.
+   Koppelvlak standaarden hebben bijvoorbeeld een kortere releasetermijn
+   dan de bovenliggende architectuur.
+11. In bijzondere gevallen kan van de releasetermijn worden afgeweken.
 12. Op het moment dat het functionele toepassingsgebied van
-Digikoppeling, waarvoor het pas-toe-of-leg-uit- regime geldt, wijzigt,
-wordt dit voorgelegd aan Forum Standaardisatie & het OBDO zodat het
-regime kan worden bekrachtigd voor dit nieuwe toepassingsgebied.
+    Digikoppeling, waarvoor het pas-toe-of-leg-uit-regime geldt
+    wijzigt, wordt dit voorgelegd aan Forum Standaardisatie en het
+    OBDO zodat het regime kan worden bekrachtigd voor dit nieuwe
+    toepassingsgebied.
+
+### Impact van wijzigingen en versienummering
+Afhankelijk van de impact van een wijziging of patch krijgt een release
+een nieuwsversienummer. Het versienummerbeheer volgt principes voor
+semantische versienummering
+[en is beschreven in een bijlage](#bijlage-versienummering-in-digikoppeling-onderdelen)
+
+De impact van een wijziging kan verschillen per koppelvlakspecificatie. Voor de
+standaarden die deel uitmaken van Digikoppeling hebben we de volgende _impactmatrix_ opgesteld:
+
+| Standaard                       | Toevoegen | Wijzigen | Verwijderen | Vervallen |
```

Comment:
tabel hoeft niet te worden opgenomen, impact is voor elk onderdeel gelijk

Reply:
Er staan een paar verschillen in voor de verschillende onderdelen.